packages/odyssey-react/src/components/Field/Field.theme.ts

```diff
@@ -16,11 +16,14 @@ export const theme: ThemeReducer = (theme) => ({
   BorderColor: "currentcolor",
   BorderStyle: "none",
   BorderWidth: 0,
-  FontMaxWidth: theme.FontLineLengthMax,
-  SpaceMarginBlockEnd: theme.SpaceRemM,
-  SpaceMarginBlockStart: 0,
-  SpaceMarginInline: 0,
-  SpacePaddingBlockEnd: theme.SpaceRemM,
-  SpacePaddingBlockStart: 0,
-  SpacePaddingInline: 0,
+  MaxWidth: theme.FontLineLengthMax,
+  MarginBlockEnd: theme.SpaceRemM,
+  MarginBlockStart: 0,
+  MarginInline: 0,
+  PaddingBlockStart: 0,
+  PaddingInline: 0,
+
+  /* eslint-disable @okta/odyssey/no-invalid-theme-properties */
```

Comment:
you can use the single line comment style for this!
```
// eslint-disable-next-line @okta/odyssey/no-invalid-theme-properties
```

no need to enable the rule again afterward in another comment